frontend/src/pages/organization/BillingPage/components/BillingCloudTab/CurrentPlanSection.tsx

```diff
@@ -48,9 +54,25 @@ export const CurrentPlanSection = () => {
               data &&
               data?.rows?.length > 0 &&
               data.rows.map(({ name, allowed, used }) => {
+                let toolTipText = null;
+                if (name === "Organization identity limit") {
+                  toolTipText = "Identity count consists of users and machine identities.";
```

Comment:
rename: Identity count is calculated by the total number of user identities and machine identities.